src/api/app/models/event_subscription/for_event_form.rb

```diff
@@ -15,8 +15,9 @@ def call
 
     private
 
+    # TODO: use back receiver_roles after the renaming of *watchers methods
```

Comment:
I don't really understand what needs to be done here. Aren't we already using `event_class.receiver_roles`?